docs/document/docs/guides/tabs-customzation.md

```diff
@@ -0,0 +1,200 @@
+---
+id: tabs-customization
+title: Tabs customization
+sidebar_label: Tabs customization
+---
+
+## Overview
+This article explains how to utilize and customize the Tabs component.<br>
+We assume the following scenario:
+1. Append KUC components to Tabs content.
+2. Append Kintone native field to Tabs content.
+
+### Components to use
+- [Tabs](../components/desktop/tabs.md)
+- [ReadOnlyTable](../components/desktop/readonly-table.md)
+
+## Completed image
+
+The completed image of the customized page is as follows:
+
+![tabs customize](assets/tabs_customize.gif)
+
+## What you will need to have ready
+
+Create an application that includes two blank fields with IDs 'space' and 'spas', and add three table fields below them with IDs 'sales_record_table' and 'task_management_table', 'sales_order_table'.
+
+## JavaScript and CSS Customization
+
+When you import the UMD file of Kintone UI Component to the app, you can upload the JavaScript files by following these steps:<br>
+You can see how to upload a file in the [Quick Start](../getting-started/quick-start.md).
+
+### Defining Constants
+
+We use constants to refer to important identifiers and values, such as the version number of Kintone, different space IDs, kintone fields, etc.
+
+```javascript
+const Kuc = Kucs['1.x.x'];
+
+const SPACE_FOR_NATIVE_KINTONE = 'space_for_native_kintone';
+const SPACE_FOR_CUSTOM_COMPONENTS = 'space_for_custom_components';
+
+const SALES_RECORD_TABLE_ID = 'sales_record_table';
+const TASK_MANAGEMENT_ID = 'task_management_table';
+const SALES_ORDER_ID = 'sales_order_table';
+
+// A set of Kintone fields that need to be shown/hidden on each tab.
+const salesRecordTabFields = [SALES_RECORD_TABLE_ID];
+const taskManagementTabFields = [TASK_MANAGEMENT_ID];
+const salesOrderTabFields = [SALES_ORDER_ID];
+let currentTabFields;
```

Comment:
Do we need this line for `#Defining constants` category?

Reply:
Please check this comment.

Reply:
I think the constants was too much.
So I create a category to describe them.

Reply:
Got it!